docs/source/developer/test_suite.rst

```diff
@@ -4,32 +4,80 @@ Test Suite
 .. contents:: Contents
   :local:
 
+
+Running the Test Suite
+==============================================================================
+
+``./csu dev test`` will run the entire test suite. For a full list of test commands available, see :doc:`../getting_started/helper_commands`.
+
+
 Structure
 ==============================================================================
 
-.. code-block:: none
+The ``tests/`` directory is in the ``csunplugged/`` directory (at the same
+level as the apps) and is structured as follows:
```

Comment:
We should be explicit stating that all tests are located here, and not within their application directories.